### ISSUE TITLE
when i try to run it locally,it seems some error happened,but no errors print , i can't get the voice

### ISSUE THREAD

**Issue by xiaoqin00** [neonbjb/tortoise-tts#28]
i have the problem in the front.it seems work,but i can't get the voice.
could u please help me ?
it troubled me serveral days.

![image](https://user-images.githubusercontent.com/13318706/167095227-698d7014-85a4-4228-ba4a-ff5a32faeca3.png)

(venv) PS C:\plan\test3\tortoise-tts-main> python tortoise/do_tts.py --text "I'm going to speak this" --voice random --preset fast
Removing weight norm...
Generating autoregressive samples..
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6/6 [00:08<00:00,  1.34s/it] 
Computing best candidates using CLVP and CVVP
  0%|                                                                                                                                                                                                               | 0/6 [00:00<?, ?it/s]C
:\plan\test3\venv\lib\site-packages\torch\utils\checkpoint.py:25: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6/6 [00:00<00:00,  6.20it/s]
Transforming autoregressive outputs into audio..
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 80/80 [00:03<00:00, 24.16it/s]

**Comment by xiaoqin00** [comment]
i try to create a test.py 。But I still can't get the sound。
`# Imports used through the rest of the notebook.
import torch
import torchaudio
import torch.nn as nn
import torch.nn.functional as F

import IPython

from tortoise.api import TextToSpeech
from tortoise.utils.audio import load_audio, load_voice, load_voices

# This will download all the models used by Tortoise from the HF hub.
tts = TextToSpeech()

# This is the text that will be spoken.
text = "Joining two modalities results in a surprising increase in generalization! What would happen if we combined them all?"

# Here's something for the poetically inclined.. (set text=)
"""
Then took the other, as just as fair,
And having perhaps the better claim,
Because it was grassy and wanted wear;
Though as for that the passing there
Had worn them really about the same,"""

# Pick a "preset mode" to determine quality. Options: {"ultra_fast", "fast" (default), "standard", "high_quality"}. See docs in api.py
preset = "fast"

# Tortoise will attempt to mimic voices you provide. It comes pre-packaged
# with some voices you might recognize.

# Let's list all the voices available. These are just some random clips I've gathered
# from the internet as well as a few voices from the training dataset.
# Feel free to add your own clips to the voices/ folder.


IPython.display.Audio('tortoise/voices/tom/1.wav')

# Pick one of the voices from the output above
voice = 'tom'

# Load it and send it through Tortoise.
voice_samples, conditioning_latents = load_voice(voice)
gen = tts.tts_with_preset(text, voice_samples=voice_samples, conditioning_latents=conditioning_latents,
                          preset=preset)
torchaudio.save('generated.wav', gen.squeeze(0).cpu(), 24000)`

**Comment by xiaoqin00** [comment]
i have tried to run it in test.py,But I still can't get the sound
![image](https://user-images.githubusercontent.com/13318706/167094563-7627220a-ca51-4b1b-ab5c-d716ccb07ae0.png)
